### DISCUSSION TITLE
How to translate audio from one language to another with timestamp aligned

### DISCUSSION THREAD

**Discussion by universeroc** [deepgram/community#342] [General help] [answered]
I find the Diarization API which can implement the speaker recognition and also recognize all audio into text.

My requirement is to make a demo to validate whether using deepgram's APIs could achieve "translate audio into other language"

This is my technology plan:

1. diarization(whisper) API to transcribe audio to text with speaker and timestamp
2. tts model api(azure etc) to translate text into audio
3. ffmpeg combine all audio files to a union timeline

If anyone has any advice please comment, thanks in advance. :)

**Comment by lukeocodes** [comment, marked as answer]
Hi @universeroc, we have a blogpost on how to achieve translations using Deepgram for transcription. I hope this helps

https://deepgram.com/learn/deepgram-universal--transcriber-translator-tutorial
